faq/serverless-sql-databases.mdx

```diff
@@ -14,7 +14,7 @@ category: serverless
 
 You are billed both for the compute resources provisioned and the storage used. You can see the cost estimate in the Database creation wizard.
 
-  - **Compute**: You are billed based on compute resources (vCPU and RAM) provisioned to handle your workload. Provisioned resources evolve according to the [autoscaling](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/#autoscaling) parameter and stay between the minimum and maximum thresholds defined for your database. For each vCPU provisioned, 4 GB of RAM will be allocated.
+  - **Compute**: You are billed based on compute resources (vCPU and RAM) provisioned to handle your workload. Provisioned resources evolve according to the [autoscaling](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/#autoscaling) parameter, and stay between the minimum and maximum thresholds defined for your database. For each vCPU provisioned, 4 GB of RAM will be allocated.
```

Comment:
```suggestion
  - **Compute**: You are billed based on compute resources (vCPU and RAM) provisioned to handle your workload. Provisioned resources evolve according to the [autoscaling](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/#autoscaling) parameter, and stay between the minimum and maximum thresholds defined for your database. For each vCPU provisioned, 4GB of RAM will be allocated.
```

Reply:
The original is right according to the style guide https://www.scaleway.com/en/docs/guidelines/#bits-and-bytes